src/System.Private.CoreLib/src/System/Reflection/Emit/ILGenerator.cs

```diff
@@ -292,42 +262,34 @@ internal __ExceptionInfo[] GetExceptions()
                 return null;
             }
 
-            temp = new __ExceptionInfo[m_exceptionCount];
+            var temp = new __ExceptionInfo[m_exceptionCount];
             Array.Copy(m_exceptions, 0, temp, 0, m_exceptionCount);
             SortExceptions(temp);
             return temp;
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Refactor this method to separate the hot path. It will make the AggressiveInlining attribute unnecessary and avoid the code bloat.

Reply:
https://github.com/dotnet/coreclr/pull/23833/commits/e55ce5573cc5e634cb91b0379cfa7e0b37f90063#diff-9251654972afbdb424e02431cb2b1e3cR264
Is this what you were thinking?

Reply:
Yes. Does this method still need AggressiveInlining after the refactoring?

Reply:
It didn't, I removed it in https://github.com/dotnet/coreclr/pull/23833/commits/b3c7e1a5f448afa738abf671ffb9d9a7a5af5ad7